lib/transactions.js

```diff
@@ -292,16 +292,16 @@ exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAdd
     }
 
     // pool fee recipients t-addr
-    if (coin.burnFees) {
-        let burn = feeReward
-        recipients.forEach(recipient => {
-            txb.addOutput(
-                scriptCompile(bitcoin.address.fromBase58Check(recipient.address).hash),
-                Math.round(blockReward * (recipient.percent / 100) - burn)
-            )
-
-            burn = 0
-        })
+    if (recipients.length > 0 && recipients[0].address != '') {
+        let burn = 0
+        if (coin.burnFees) {
+            burn = feeReward
+        }
+        recipients.forEach(recipient => txb.addOutput(
+            scriptCompile(bitcoin.address.fromBase58Check(recipient.address).hash),
+            Math.round(blockReward.total * (recipient.percent / 100) - burn)
+        ))
+        burn = 0
```

Comment:
I think this needs to stay inside the loop because it only Burns the first one?

Reply:
@egyptianbman made changes and tested. s-nomp starting fine like this.